modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/store/actions/handleFieldBlurred.es.js

```diff
@@ -22,9 +22,15 @@ export default (pages, properties) => {
 		pageVisitor.mapFields(field => {
 			const matches = field.name === fieldInstance.name;
 
+			let displayErrors = false;
+
+			if (matches && field.required && fieldInstance.value == '') {
```

Comment:
@linolaoj Here you are only accounting for required validations. What about other kind of validations set by the user? I think displayErrors will be false for all other validations.

Reply:
Hi @brunobasto , the idea was to let displayErrors as false until the evaluation is done and we have the correct validation result. I've added the validation for required fields because it is a case that it is easy to validate and should not change after validation even when using a "required" rule.

Reply:
Got it. Makes sense then.